x-pack/packages/kbn-elastic-assistant/impl/assistant_context/index.tsx

```diff
@@ -202,11 +225,15 @@ export const AssistantProvider: React.FC<AssistantProviderProps> = ({
       augmentMessageCodeBlocks,
       allQuickPrompts: localStorageQuickPrompts ?? [],
       allSystemPrompts: localStorageSystemPrompts ?? [],
+      baseAllow: uniq(baseAllow),
```

Comment:
Ty! Will need similar sanitization/validation on the system/quick prompts side as well 👍